library/Ivoz/Provider/Domain/Service/RoutingPatternGroup/UpdateByRoutingPatternAndCountry.php

```diff
@@ -46,10 +46,7 @@ public function __construct(
     public function execute(RoutingPatternInterface $entity, CountryInterface $country)
     {
 
-        if (!$country->getZone()) {
-            /**
-             * @todo Every country has a value on db. Is this even needed?
-             */
+        if (!$country->getZone()->getEn()) {
```

Comment:
I don't think this condition is possible because Countries is a read-only table with fixed data.

Reply:
Removed